source/deploy/mobile-app-changelog.md

```diff
@@ -2,12 +2,48 @@
 
 Latest Mattermost Mobile Apps releases:
 
-- [2.3.0 Release](#release)
-- [2.2.0 Release](#id1)
-- [2.1.0 Release](#id6)
-- [2.0.1 Release](#id12)
-- [2.0.0 Release](#id17)
-- [1.55.1 Release](#id20)
+- [2.4.0 Release](#release)
+- [2.3.0 Release](#id1)
+- [2.2.0 Release](#id7)
+- [2.1.0 Release](#id12)
+- [2.0.1 Release](#id18)
+- [2.0.0 Release](#id23)
+
+## 2.4.0 Release
+- Release Date: May 16, 2023
+- Server Versions Supported: Server v7.8.0+ is required. Self-Signed SSL Certificates are not supported unless the user installs the CA certificate on their device.
+
+### Compatibility
+ - **Upgrade to server version v7.8.0 or later is required.** Support for server [Extended Support Release](https://docs.mattermost.com/upgrade/extended-support-release.html) (ESR) v7.1.0 has ended and upgrading to server ESR v7.8.0 or later is required. As we innovate and offer newer versions of our mobile apps, we maintain backwards compatibility only with supported server versions. Users who upgrade to the newest mobile apps while being connected to an unsupported server version can be exposed to compatibility issues, which can cause crashes or severe bugs that break core functionality of the app.
+ - Android operating system 7+ [is required by Google](https://android-developers.googleblog.com/2017/12/improving-app-security-and-performance.html).	
+ - iPhone 5s devices and later with iOS 12.1+ is required.
+
+### Improvements
+ - Added mobile support for message acknowledgements and persistent notifications.
+ - Added the "Add members" screen.
+ - Increased the character limit in the nickname field in the user interface from 22 to 64 characters.
+ - Added localization support for iOS usage description strings in permission dialogs.
+ - Users can now follow or unfollow a thread and undo the action if needed.
+ - Added a channel files screen.
+ - Added support for ``ExperimentalSettings.DelayChannelAutocomplete``.
+ - Calls: Call threads are now auto-followed when joining from mobile.
+ - Calls, Android: Fixed an issue with bluetooth; added audio device selection menu.
+ - Added support for self-hosted Sentry URL parameters in build scripts.
+ - Added a ``files_count`` column to the ``ChannelInfo`` table and database migration.
```

Comment:
This one is more or less part of the effort on line 27: `Added a channel files screen.`. 0/5 on whether we should delete this line and just keep the one on line 27.

Reply:
Thanks, I removed line 32.